tests/integration/012_environment/main.tf

```diff
@@ -16,12 +16,17 @@ resource "env0_environment" "example" {
   name          = "environment"
   project_id    = env0_project.test_project.id
   template_id   = env0_template.template.id
+  wait_for      = "FULLY_DEPLOYED"
   configuration {
     name  = "environment configuration variable"
     value = "value"
   }
   approve_plan_automatically = true
-  revision                   = "master"
```

Comment:
Changed the integration test back to what it used to be, minus the `time_sleep` to wait for deployment to finish